index.js

```diff
@@ -0,0 +1,44 @@
+
+
+// First exercise --------------------
+const countMovesUp = inputValue => inputValue.match(/\(/g).length
+
+const countMovesDown = inputValue => inputValue.match(/\)/g).length
+
+const santasFloor = inputValue => countMovesUp(inputValue) - countMovesDown(inputValue)
+
+
+// Second exercise --------------------
+const santasFirstPositionAtSubsolo = ( position, santasFlorPosition) => {
+  let i = position;
+
+  for ( i; i < inputValue.length; i++ ) {
+    if ( inputValue[i] == '(' ) {
+      santasFlorPosition += 1
+    } else {
+      santasFlorPosition -= 1
+      if ( santasFlorPosition == -1 ) {
+        return i + 1
+      }
+    }
+  }
+}
+
+
+// Answers --------------------
+const elementAnswer = document.getElementById('answer')
+const firstAnswer = santasFloor(inputValue)
+const secondAnswer = santasFirstPositionAtSubsolo(0, 0)
+
+function createMarkup(answer, exerciseNumber) {
+  return (`
+    <h2>Answer ${exerciseNumber} is ${answer}</h2>
+  `);
+}
+
+
+elementAnswer.innerHTML = createMarkup(firstAnswer, '1') + createMarkup(secondAnswer, '2')
```

Comment:
To make the test works maybe you have to encapsulate this inside a function and call it from html code.
Then you can export functions below.